.github/workflows/Windows.yml

```diff
@@ -130,6 +130,24 @@ jobs:
       if: ${{ failure() }}
       shell: bash
       run: cat ODBC_TRACE.log
+      
+    - name: check out duckdbDotNetOdbcTests
+      uses: actions/checkout@v4
+      with:
+        repository: bucweat/duckdbDotNetOdbcTests
+        path: duckdbDotNetOdbcTests
```

Comment:
Instead of cloning an external repository could we move the .NET tests to `tools/odbc/tests/dotnet`

Reply:
> Instead of cloning an external repository could we move the .NET tests to `tools/odbc/tests/dotnet`

Done :-)